### PR TITLE
Add cargo deb and fix espflash version

### PR DESCRIPTION
**Pull request by diekleinekuh**

### CODE REVIEW

File: Dockerfile.template [hunk 1]
```diff
@@ -17,7 +17,11 @@ RUN rustup target add aarch64-unknown-linux-gnu
 RUN rustup target add aarch64-unknown-linux-musl
 
 RUN curl -L https://github.com/ryankurte/cargo-binstall/releases/latest/download/cargo-binstall-x86_64-unknown-linux-musl.tgz | tar xz -C ~/.cargo/bin
-RUN cargo binstall --no-confirm espflash
+RUN cargo binstall --no-confirm espflash --version 1.5.1
```
Comment (andresv) [inline Dockerfile.template]: 1.7 is out with maximes fixes

Comment (diekleinekuh) [reply Dockerfile.template]: I just wanted to add something to the image to produce debs. But it seems like even without cargo binstall always the newest version of espflash is installed which breaks current build scripts as they suddenly require to add a CHIP parameter. 
Also I'm not really sure about binstall as it seems to fail to make installed binaries executable. I don't see the point of using it while creating the image when we could just use cargo install. On top of that we installed espflash while we actually use cargo-espflash. I need to have a closer look on Monday.
In any case we should push a new image with a new tag to not break anything. Then we can upgrade to newest espflash and fix ci scripts in a branch

File: 1.61/cargo-deb/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# vim:set ft=dockerfile:
+
+# Do not edit individual Dockerfiles manually. Instead, please make changes to the Dockerfile.template, which will be used by the build script to generate Dockerfiles.
+
+FROM cimg/rust:1.61.0
+
+LABEL maintainer="Maxime Borges <maxiem@yaak.ai>"
```
Comment (andresv) [inline 1.61/cargo-deb/Dockerfile]: maxime@yaak.ai

Comment (diekleinekuh) [reply 1.61/cargo-deb/Dockerfile]: Maybe it's spam protection?

Comment (maximeborges) [reply 1.61/cargo-deb/Dockerfile]: Involuntary spam protection

File: 1.61/cargo-deb/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+# Also configure to not install any extra packages as they are pulled in at the latest version which is not desireable for reproducability.
+RUN curl -LO https://github.com/esp-rs/rust-build/releases/download/v${RUST_VERSION}.0/install-rust-toolchain.sh && \
+ 	chmod a+x install-rust-toolchain.sh && \
+ 	./install-rust-toolchain.sh --toolchain-version ${RUST_VERSION}.0 --extra-crates "" --clear-cache "YES" && \
+	rustup toolchain uninstall stable && \
+	rustup toolchain uninstall nightly && \
+	rm install-rust-toolchain.sh
+
+RUN rustup target add aarch64-unknown-linux-gnu
+RUN rustup target add aarch64-unknown-linux-musl
```
Comment (andresv) [inline 1.61/cargo-deb/Dockerfile]: are both needed?

Comment (diekleinekuh) [reply 1.61/cargo-deb/Dockerfile]: No, will remove.

**Review by andresv — APPROVED** [review]
Let's try out then.

**Pull request merged**